Kiosk/Views/SettingsPage.xaml

```diff
@@ -143,6 +143,27 @@
 
                     <PasswordBox Header="Translator Text API Key" Password="{Binding TranslatorTextApiKey, Mode=TwoWay}" Margin="0,10,0,0" />
 
+                    <Grid Margin="0,10,0,0">
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition/>
+                            <ColumnDefinition Width="auto"/>
+                            <ColumnDefinition Width="auto"/>
+                        </Grid.ColumnDefinitions>
+                        <PasswordBox Header="Ink Recognizer API Key" Password="{Binding InkRecognizerApiKey, Mode=TwoWay}" />
+                        <ComboBox Grid.Column="1" Margin="10,0,6,0" Header="Endpoint" Width="355"
+                                  x:Name="inkRecognizerEndpointCombobox"
+                                  ItemsSource="{Binding InkRecognizerEndpoints}"
+                                  SelectedItem="{Binding BindingInkRecognizerApiKeyEndpoint, Mode=TwoWay}">
+                        </ComboBox>
+                        <StackPanel Grid.Column="2" Orientation="Horizontal" 
```

Comment:
As mentioned in the other comment, the only part needed here will be the API Key.

Reply:
Changed to only include a PasswordBox for the Ink Recognizer API key